ignore/src/walk.rs

```diff
@@ -416,6 +427,17 @@ impl DirEntryRaw {
             ino: md.ino(),
         })
     }
+
+    // Placeholder implementation to allow compiling on non-standard platforms (e.g. wasm32).
+    #[cfg(not(any(windows, unix)))]
+    fn from_path(
+        depth: usize,
+        pb: PathBuf,
+        link: bool,
+    ) -> Result<DirEntryRaw, Error> {
+        Err(Error::Io(io::Error::new(
+            io::ErrorKind::NotFound, "unsupported platform")))
```

Comment:
I don't think `NotFound` is the right error kind for this. Can you use `Other` instead?

Reply:
Done.